test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1

```diff
@@ -16,7 +16,7 @@ Describe "Experimental Feature Basic Tests - Feature-Disabled" -tags "CI" {
             $TestModule = Join-Path $PSScriptRoot "assets" "ExpTest"
             $AssemblyPath = Join-Path $TestModule "ExpTest.dll"
             if (-not (Test-Path $AssemblyPath)) {
-                ## When using $SourcePath directly, 'Add-Type' fails in AppVeyor CI runs with an 'access denied' error.
+                ## When using $SourcePath directly, 'Add-Type' fails in Windows CI runs with an 'access denied' error.
```

Comment:
I think this comment should NOT be changed, since it was an AppVeyor specific issue. One should check if the problem still persist on Azure DevOps build infrastructure. Maybe even open a new issue to track this work.